static/styles/settings.css

```diff
@@ -177,6 +177,11 @@ h3,
     width: 100%;
     max-width: 500px;
     box-sizing: border-box;
+
+    &:disabled {
+        cursor: not-allowed;
+        background-color: hsl(0, 0%, 93%);
+    }
```

Comment:
Added it here because only the realm description setting can be disabled, but moved it to `settings_textarea` since we may add such textarea elements in future which can be disabled.

Reply:
Yeah that seems correct.